kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -189,6 +189,13 @@ data:
     # [agent_settings.proxy_config]
     #    ignore_proxy_settings = "true"  # if this is not applied, default value is false
 
+    # The following settings are "undocumented", we don't recommend uncommenting them unless directed by Microsoft.
+    # Configuration settings for the waittime for the network listeners to be available
+    # [agent_settings.network_listener_waittime] 
+    #   tcp_port_25226 = 45                           # Port 25226 is used by the fluent-bit in ReplicaSet
```

Comment:
probable, we can improve the comment saying for example from "Port 25226 is used by the fluent-bit in ReplicaSet"  to "Port 25226 is used for telegraf to fluent-bit in ReplicaSet" or something like this being used between telegraf to fluent-bit/fluentd etc.